src/components/views/context_menus/WidgetContextMenu.tsx

```diff
@@ -48,6 +49,69 @@ interface IProps extends Omit<ComponentProps<typeof IconizedContextMenu>, "child
     onEditClick?(): void;
 }
 
+const showStreamAudioStreamButton = (app: IWidget): boolean => {
```

Comment:
Can we add some doc comments where relevant here?
It sometimes has logic that is not straightforward to follow